tests/RoutingTest.php

```diff
@@ -118,7 +118,7 @@ public function injects_route_placeholder_over_request_attribute()
     public function resolve_controller_from_container()
     {
         $app = Bridge::create();
-        $app->get('/', [UserController::class, 'dashboard']);
```

Comment:
Thanks for the PR! This is a regression here, this shouldn't be changed.

This is because `AppFactory::createFromContainer($container)` doesn't work like the old method. `AppFactory::setCallableResolver(new CallableResolver($callableResolver));` should be removed, and instead the callable resolver should be defined in the container.

Reply:
Thanks for pointing this out! I updated the PR.